recipe/meta.yaml

```diff
@@ -1,57 +1,45 @@
-{% set version = "1.1.1l" %}
+{% set version = "3.0.0" %}
 
 package:
   name: openssl_split
   version: {{ version }}
 
 source:
   url: http://www.openssl.org/source/openssl-{{ version }}.tar.gz
-  sha256: 0b7a3e5e59c34827fe0c3a74b7ec8baef302b98fa80088d7f9153aa16fa76bd1
+  sha256: 59eedfcb46c25214c9bd37ed6078297b4df01d012267fe9e9eee31f61bc70536
   patches:
-    - patches/0001-Don-t-use-USE_BCRYPTGENRANDOM-for-VS-older-than-2015.patch
-    # backport of https://github.com/openssl/openssl/pull/16409; can be dropped for ver >1.1.1l
-    - patches/0002-Darwin-platform-allows-to-build-on-releases-before-Y.patch
+    # backport from openssl/openssl#16593 to simplify & future-proof build;
+    # can be dropped for any version after 3.0.0
+    - patches/0001-80-test_cmp_http.t-Fix-handling-of-empty-HTTP-proxy-.patch
 
 build:
   number: 0
-  ignore_run_exports_from:
-    # For "perl" in "requirements/host" on windows, see below.
-    - perl *
 
 requirements:
   build:
     - {{ compiler('c') }}
     - nasm               # [win]
     - make               # [unix]
-    - perl *             # [not win]
+    - perl
```

Comment:
```suggestion
    - perl *
```

Reply:
Is there a functional difference between the two (or anything worth indicating with the `*`)? If yes, I was unaware & can re-add of course.

Reply:
Yes, if there's no `*`, conda-build will generate multiple jobs if there was multiple versions for `perl` in conda-forge pinning.

Reply:
Ah, makes sense, thx.